### PR TITLE
Fix build without PG_SUPPORT

### PR DESCRIPTION
**Pull request by alekece**
This PR aims to resolve a new bug occuring since the backport of PostgreSQL - quick and easy.

### CODE REVIEW

File: ledger-core/CMakeLists.txt [hunk 1]
```diff
@@ -31,8 +31,33 @@ set(CMAKE_MACOSX_RPATH 1)
...
+    ledger-qt-host
+    ledger-core-integration-test
+    mongoose
+    ssl
+    gtest gtest_main
+)
+
 if (PG_SUPPORT)
     add_definitions("-DPG_SUPPORT")
+else()
```
Comment (phaazon) [inline ledger-core/CMakeLists.txt]: I’m not sure this is the right way to do. That seems odd.

Comment (gagbo) [reply ledger-core/CMakeLists.txt]: I would prefer adding to the list in the `if (PG_SUPPORT)` branch too.

File: ledger-core/CMakeLists.txt [hunk 2]
```diff
@@ -31,6 +31,10 @@ set(CMAKE_MACOSX_RPATH 1)
 
 add_definitions("-DSQLITE_HAS_CODEC")
 
+set(INSTALL_TARGETS
+
+)
+
```
Comment (gagbo) [inline ledger-core/CMakeLists.txt]: What's the purpose of this variable ? Does it get filled automatically when calling `install` later ?

Comment (alekece) [reply ledger-core/CMakeLists.txt]: Good catch ! I will remove it :trollface:

**Pull request merged**